ssaEdge/eclipe-thingweb/thing-model/src/tm-helpers.ts

```
@@ -0,0 +1,712 @@
+   *
+   * @experimental
+   */
+  public async getPartialTDs(
+    model: unknown,
+    options?: CompositionOptions,
+  ): Promise<ExposedThingInit[]> {
+    const extendedModels = await this._getPartialTDs(model, options);
+    const extendedPartialTDs = extendedModels.map((_data) => {
+      const data = _data as ExposedThingInit;
+      // change the @type
+      if (data["@type"] instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)